### PR TITLE
Set 'input.disabled=true' should raise the `change` event in Chrome (closes #2240)

### LINKED ISSUE #2240: Set 'input.disabled=true' should emit the `change` event in Chrome
This issue is related to https://github.com/DevExpress/testcafe/issues/4736

### PR DESCRIPTION
**Pull request by AlexKamaev**

### CODE REVIEW

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

File: src/client/sandbox/event/element-editing-watcher.ts [hunk 1]
```diff
@@ -39,10 +39,10 @@ export default class ElementEditingWatcher {
             nativeMethods.removeEventListener.call(el, 'blur', e => this._onBlur(e));
             nativeMethods.removeEventListener.call(el, 'change', e => this._onChange(e));
 
-            if (el[ELEMENT_EDITING_OBSERVED_FLAG])
+            if (el[ELEMENT_EDITING_OBSERVED_FLAG] !== void 0)
```
Comment (LavrovArtem) [inline src/client/sandbox/event/element-editing-watcher.ts]: Maybe use the `hasOwnProperty`?

File: src/client/sandbox/node/window.ts [hunk 2]
```diff
@@ -883,6 +886,29 @@ export default class WindowSandbox extends SandboxBase {
             }
         });
 
+        // NOTE: HTMLInputElement raises the `change` event on `disabled` only in Chrome
+        if (isChrome) {
+            overrideDescriptor(window.HTMLInputElement.prototype, 'disabled', {
+                getter: function () {
+                    return nativeMethods.inputDisabledGetter.call(this);
+                },
```
Comment (LavrovArtem) [inline src/client/sandbox/node/window.ts]:
```js
getter: null,
```
This code stays the native getter as is.

**Pull request merged**